vendor/github.com/openshift/library-go/pkg/operator/staleconditions/remove_stale_conditions.go

```diff
@@ -25,7 +25,7 @@ func NewRemoveStaleConditionsController(
 		conditions:     conditions,
 		operatorClient: operatorClient,
 	}
-	return factory.New().ResyncEvery(time.Second).WithSync(c.sync).WithInformers(operatorClient.Informer()).ToController("RemoveStaleConditionsController", eventRecorder.WithComponentSuffix("remove-stale-conditions"))
+	return factory.New().ResyncEvery(time.Minute).WithSync(c.sync).WithInformers(operatorClient.Informer()).ToController("RemoveStaleConditionsController", eventRecorder.WithComponentSuffix("remove-stale-conditions"))
```

Comment:
I dunno why this is coming in with my bump, but it's from openshift/library-go#1036.  Maybe a previous commit rev'ed the pinned library-go commit, but forgot to update this vendored file?

Reply:
Seems like it was #522:

```console
$ git show 24b7188b30319aabb1621062f25750b311bb5688 vendor/modules.txt | grep '#.*library-go'
-# github.com/openshift/library-go v0.0.0-20201123212217-43f358922ea0
+# github.com/openshift/library-go v0.0.0-20210330121117-68dd4a4c9d9e
```